AUTHORS

```diff
@@ -30,3 +30,4 @@ Contributors
 * Priyabrata Mondal
 * Farouk Ech-Charef
 * Mustapha Belbiad
+* VaradRengaraj
```

Comment:
The authors' names should be listed as 
`First Last`
with a whitespace in between. Is there a reason for not using the whitespace?

Reply:
Possibly relevant: https://github.com/pyccel/pyccel/pull/1762#issuecomment-1988299977

Reply:
I do not agree with using the GitHub username. The AUTHORS file is there to clearly identify who has the copyright over the code.

Reply:
Although one could argue that first and last name are not necessarily sufficient for that... 😅

Reply:
@EmilyBourne No, please do it.

Reply:
Would you mind doing it. I assume that your name is "Varad Regaraj" but I don't actually know if this is your name or just your GitHub username

Reply:
ok I will do it.